migration_toolkit_for_containers/installing-mtc.adoc

```diff
@@ -8,7 +8,10 @@ toc::[]
 
 You can install the {mtc-full} ({mtc-short}) on {product-title} 4.
 
-After you manually install the legacy {mtc-full} Operator on the {product-title} 4 source cluster, you install the {mtc-full} Operator on the {product-title} {product-version} target cluster by using the Operator Lifecycle Manager.
+[NOTE]
+====
+To install {mtc-short} on {product-title} 3, see xref:../migrating_from_ocp_3_to_4/installing-3-4.adoc#migration-installing-legacy-operator_installing-3-4[Installing the legacy {mtc-full} Operator on {product-title} 3].
+====
```

Comment:
Per the OpenShift glossary of terms, `[containers](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#container)` should not be capitalized. If time permits, please update the heading.